.github/workflows/issues_bot.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Automatically close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-issue-message: 'We are clearing up our old issues and your ticket has been open for one year with no activity. Remove stale label or comment or this will be closed in 15 days.'
+          days-before-stale: 365
+          days-before-close: 15
+          exempt-issue-labels: 'enhancement,feature-request,investigating,needs-work,pending-deploytriaged,'
```

Comment:
Nit: Can you please a new line?

<img width="188" alt="Screen Shot 2021-04-22 at 9 35 58 AM" src="https://user-images.githubusercontent.com/2487951/115751561-25b00000-a34e-11eb-9cce-8b2a7e44c893.png">

Reply:
Fixed!